cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -73,11 +73,11 @@ data:
   # of the CSI driver to something other than what is officially supported, change
   # these images to the desired release of the CSI driver.
   # ROOK_CSI_CEPH_IMAGE: "quay.io/cephcsi/cephcsi:v3.3.1"
-  # ROOK_CSI_REGISTRAR_IMAGE: "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1"
-  # ROOK_CSI_RESIZER_IMAGE: "k8s.gcr.io/sig-storage/csi-resizer:v1.0.1"
-  # ROOK_CSI_PROVISIONER_IMAGE: "k8s.gcr.io/sig-storage/csi-provisioner:v2.0.4"
-  # ROOK_CSI_SNAPSHOTTER_IMAGE: "k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0"
-  # ROOK_CSI_ATTACHER_IMAGE: "k8s.gcr.io/sig-storage/csi-attacher:v3.0.2"
+  # ROOK_CSI_REGISTRAR_IMAGE: "quay.io/k8scsi/csi-node-driver-registrar:v2.0.1"
```

Comment:
@tristan-tsl 
- There are several places that would need to be updated in addition to this, if you search all of rook.
- Do you have background for quay.io becoming the preferred location for the images? @Madhu-1 wdyt?

Reply:
AFAIK Kubernetes CSI is not pushing the images to quay.io anymore. the latest images are getting pushed to gcr.

https://github.com/kubernetes-csi/csi-release-tools/blob/682c686aeb78b5390c945d7b8feb6c927cc1feff/cloudbuild.yaml#L35